React Project/src/pages/ProjectPage/ProjectPage.tsx

```diff
@@ -1,43 +1,29 @@
 import { useLocation } from "react-router-dom";
-import { Project } from "../ProjectsList/ProjectList";
+import { Project } from "../../lib/types";
 
 export default function ProjectPage() {
     let { state }: { state: Project } = useLocation();
```

Comment:
Instead of passing the Project in State, lets create an endpoint to fetch project by Id and use that here.

Reply:
https://trello.com/c/kdY1iWKU

Reply:
On it.